grunt/tasks/less.js

```diff
@@ -33,28 +34,29 @@ module.exports = function(grunt) {
       // Check to see if the screen size value is larger than the em threshold
       // If value is larger than em threshold, convert value (assumed px) to ems
       // Otherwise assume value is in ems
-      var largeEmBreakpoint = screenSize.large > screensizeEmThreshold ?
-        screenSize.large / baseFontSize :
-        screenSize.large;
-      var mediumEmBreakpoint = screenSize.medium > screensizeEmThreshold ?
-        screenSize.medium / baseFontSize :
-        screenSize.medium;
-      var smallEmBreakpoint = screenSize.small > screensizeEmThreshold ?
-        screenSize.small / baseFontSize :
-        screenSize.small;
+      var largeEmBreakpoint = screenSize.large > screensizeEmThreshold
+        ? screenSize.large / baseFontSize
+        : screenSize.large;
+      var mediumEmBreakpoint = screenSize.medium > screensizeEmThreshold
+        ? screenSize.medium / baseFontSize
+        : screenSize.medium;
+      var smallEmBreakpoint = screenSize.small > screensizeEmThreshold
+        ? screenSize.small / baseFontSize
+        : screenSize.small;
 
       imports += `\n@adapt-device-large: ${largeEmBreakpoint}em;`;
       imports += `\n@adapt-device-medium: ${mediumEmBreakpoint}em;`;
       imports += `\n@adapt-device-small: ${smallEmBreakpoint}em;\n`;
     }
 
     if (options.mandatory) {
+      // eslint-disable-next-line no-redeclare
```

Comment:
would be preferable to switch the `for` loop counter to be declared with `let` rather than use this eslint directive (same goes for line 69)